hasher-matcher-actioner/webapp/src/Sidebar.tsx

```diff
@@ -47,6 +47,14 @@ export default function Sidebar(
             Submit Content
           </NavLink>
         </li>
+        <li className="nav-item">
```

Comment:
suggestion: leave off of sidebar until ready/connected to system like I did with the submit page, it can still be access using `/banks/`